pkg/auth/login.go

```diff
@@ -48,6 +51,13 @@ func Login(server, username, password, token, caAuth string, skipTLS bool) error
 	if err != nil {
 		return err
 	}
-
+	// Process the messages returned by openshift login code and print our message
+	originalOutMsg := loginOutBuffer.Bytes()
+	var extraOut string
+	// If new-project is present in original message, then ask user to do the same in our message
+	if bytes.Contains(originalOutMsg, []byte("new-project")) {
+		extraOut = " No project? create one\n\todo project create <projectname>"
```

Comment:
I would put this on a different line (just prepend `\n` to `extraOut` should do the trick)…

Reply:
Any case i have found a way to work around by separating `extraOut` completely. Its closer now to Openshift messaging without being the same thing

```
./odo login -u developer
 ✓  Logged in to "https://192.168.42.114:8443" as "developer"

You don't have any projects. You can try to create a new project, by running

	odo project create <projectname>


```